addons/binding/org.openhab.binding.elerotransmitterstick/src/main/java/org/openhab/binding/elerotransmitterstick/handler/EleroTransmitterStickHandler.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.elerotransmitterstick.handler;
+
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseBridgeHandler;
+import org.eclipse.smarthome.core.thing.binding.BridgeHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.elerotransmitterstick.config.EleroTransmitterStickConfig;
+import org.openhab.binding.elerotransmitterstick.stick.TransmitterStick;
+import org.openhab.binding.elerotransmitterstick.stick.TransmitterStick.StickListener;
+
+/**
+ * The {@link EleroTransmitterStickHandler} is responsible for managing the connection to an elero transmitter stick.
+ *
+ * @author Volker Bier - Initial contribution
+ */
+public class EleroTransmitterStickHandler extends BaseBridgeHandler implements BridgeHandler {
+    private TransmitterStick stick;
+
+    public EleroTransmitterStickHandler(Bridge bridge) {
+        super(bridge);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUid, Command command) {
```

Comment:
Why do you override it if you do not implement it?

Reply:
Because it is part of the interface.